js/plugins/iq.js

```diff
@@ -0,0 +1,348 @@
+require("../framework/InitAnalyticsNamespace.js");
+require("../../html5-common/js/utils/utils.js");
+
+/**
+ * @class IqPlugin
+ * @classdesc This is an example class of a plugin that works with the Ooyala Analytics Framework.
```

Comment:
Can you fix the class description? It's still reference the stuff from our template class.

Reply:
Done